docs/tutorials/work_with_lsdb.ipynb

```diff
@@ -7,7 +7,9 @@
    "source": [
```

Comment:
<p>Also, can we add a quiet flag to this call? It has a lot of output on readthedocs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-dask/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>